### DISCUSSION TITLE
Can information about chromatin accessibility around a gene be inferred from the Browser Track plots?

### DISCUSSION THREAD

**Discussion by Kaivalyamolugu** [GreenleafLab/ArchR#836] [General]
Hello, 
This question is related to the Issue #626. I have played with different set.seed values and different sizes of plotting windows and noticed that the browser track plots change depending on the number that is given to set.seed. I'm wondering if any inference about chromatin accessibility can be made at all based on these plots considering they change significantly based on the set.seed number? Is there any other way to quantify and compare chromatin accessibility across samples around a gene of interest?

Thanks, 
Kaivalya

**Comment by rcorces** [comment]
These plots are illustrative and you should use robust statistical tests to identify important regions rather than visual inspection of tracks

**Reply by Kaivalyamolugu** [reply]
I am trying to compare chromatin accessibilities of certain genes across my samples. Is there a way of doing statistical analysis just around these genes using ArchR even though they don't pull as the top differential regions?

**Reply by rcorces** [reply]
You can subset matrices etc or you can export matrices as summarized experiments and perform your own tests. ArchR does not provide such customized features natively.
